vpdq/setup.cfg

```diff
@@ -0,0 +1,4 @@
+[metadata]
```

Comment:
What's the advantage of splitting the configuration between setup.cfg and setup.py vs just putting it all in setup.py, or all in setup.cfg and using the dummy setup()?